components/centraldashboard/public/components/notebooks-default-card.js

```diff
@@ -0,0 +1,100 @@
+import '@polymer/iron-ajax/iron-ajax.js';
+import '@polymer/iron-icon/iron-icon.js';
+import '@polymer/iron-icons/iron-icons.js';
+import '@polymer/paper-card/paper-card.js';
+import '@polymer/paper-item/paper-icon-item.js';
+import '@polymer/paper-ripple/paper-ripple.js';
+import '@polymer/paper-icon-button/paper-icon-button.js';
+import '@polymer/paper-progress/paper-progress.js';
+import './iframe-link.js';
+// eslint-disable-next-line max-len
+import {AppLocalizeBehavior} from '@polymer/app-localize-behavior/app-localize-behavior.js';
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+
+import {html, PolymerElement} from '@polymer/polymer';
+
+import './card-styles.js';
+
+/**
+ * Component to retrieve and display the default notebook
+ */
+// eslint-disable-next-line max-len
+export class DefaultNotebookCard extends mixinBehaviors([AppLocalizeBehavior], PolymerElement) {
```

Comment:
Class name should be NotebookDefaultCard if the file is now notebooks-default-card.js (REally minor thing i know, but just flagging it to assure our naming consistency)

Reply:
yup, sorry